ooni/settings.py

```diff
@@ -325,6 +326,20 @@ def web_ui_url(self):
             self.advanced.webui_port
         )
 
+    @property
+    def platform(self):
+        if os.path.exists('/etc/default/lepidopter'):
+            return 'lepidopter'
+        system = platform.system()
+        if system == 'Darwin':
+            return 'macos'
+        elif system == 'Linux':
+            return 'linux'
+        elif system == 'Windows':
+            # Really?
+            return 'windows'
+        return 'unknown'
```

Comment:
Why don't you want to return `platform.system().lower()` instead of `'unknown'`?